### PR TITLE
:wrench: update export-rates-template spec to new format: request-rates-template and download-rates-template

### PR DESCRIPTION
**Pull request by yoerriwalstra**
Related Jira Issue: https://myparcelcombv.atlassian.net/browse/MP-5988

### CODE REVIEW

File: specification/paths/RequestRatesTemplate.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "contracts.manage"
+        ]
+      }
+    ],
+    "summary": "Request rates template export.",
+    "description": "This endpoint queues a job to create a rates template (CSV) based on existing service rates. The template can be downloaded on completion.",
```
Comment (NickVries) [inline specification/paths/RequestRatesTemplate.json:14]:
```suggestion
    "description": "This endpoint queues a job to create a rates template (CSV) based on existing service rates. The template can be downloaded on completion using the `/download-rates-template` endpoint.",
```

**Review by NickVries — CHANGES_REQUESTED** [review]
One more thing, I think it'd be good to have some kind of information on either of the endpoints (I'm currently not sure which one it is) to indicate how to check the progress of the file download. It's returned by one of the endpoints, right?

File: specification/paths/DownloadRatesTemplate.json [hunk 2]
```diff
@@ -10,10 +10,10 @@
         ]
       }
     ],
-    "summary": "Export service rates template.",
-    "description": "This endpoint retrieves a template based on existing service rates, this can then be used to edit existing and add new service rates into the system.",
+    "summary": "Download rates template export.",
+    "description": "This endpoint retrieves a rates template (CSV), this can then be used to edit existing and add new service rates into the system.",
     "requestBody": {
-      "description": "The object containing a contract id and optional service codes.",
+      "description": "The object containing a request id.",
```
[the remark this answers is not in the document]
Comment (yoerriwalstra) [reply specification/paths/DownloadRatesTemplate.json]: done

File: specification/paths/RequestRatesTemplate.json (hunk 1 above)
Comment (yoerriwalstra) [reply specification/paths/RequestRatesTemplate.json]: done

**Review by M4tini — APPROVED** [review]

**Pull request merged**